Resources/Prototypes/DeltaV/Voice/speech_emote_sounds.yml

```diff
@@ -116,6 +118,8 @@
       collection: VulpkaninWhines
     Howl:
       collection: VulpkaninHowls
+    Awoo:
```

Comment:
New emote spotted, add it to the Harpy List.

Reply:
Otherwise this is fine. :)

Reply:
if i’m not mistaken this also makes harpies able to awoo, the commit also adds to the HarpySounds